[2023-10-17T11:19:12.175+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.185+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.185+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:12.203+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-14 00:00:00+00:00
[2023-10-17T11:19:12.210+0000] {standard_task_runner.py:57} INFO - Started process 571 to run task
[2023-10-17T11:19:12.213+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-14T00:00:00+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp2aa81j7j']
[2023-10-17T11:19:12.214+0000] {standard_task_runner.py:85} INFO - Job 77: Subtask waiting_for_data
[2023-10-17T11:19:12.257+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:12.335+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-14T00:00:00+00:00'
[2023-10-17T11:19:12.347+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:12.347+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:12.347+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:12.348+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:12.357+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210314T000000, start_date=20231017T111912, end_date=20231017T111912
[2023-10-17T11:19:12.388+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:12.405+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:41.127+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-18T10:42:41.137+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-18T10:42:41.138+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:41.153+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-14 00:00:00+00:00
[2023-10-18T10:42:41.160+0000] {standard_task_runner.py:57} INFO - Started process 1535 to run task
[2023-10-18T10:42:41.163+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-14T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpuqvhfmkp']
[2023-10-18T10:42:41.165+0000] {standard_task_runner.py:85} INFO - Job 76: Subtask waiting_for_data
[2023-10-18T10:42:41.216+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:41.292+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-14T00:00:00+00:00'
[2023-10-18T10:42:41.312+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:41.316+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210314T000000, start_date=20231018T104241, end_date=20231018T104241
[2023-10-18T10:42:41.328+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 76 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1535)
[2023-10-18T10:42:41.375+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:41.421+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:11.318+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-18T11:17:11.336+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-18T11:17:11.336+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:11.358+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-14 00:00:00+00:00
[2023-10-18T11:17:11.366+0000] {standard_task_runner.py:57} INFO - Started process 343 to run task
[2023-10-18T11:17:11.370+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-14T00:00:00+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpudo641wq']
[2023-10-18T11:17:11.371+0000] {standard_task_runner.py:85} INFO - Job 105: Subtask waiting_for_data
[2023-10-18T11:17:11.429+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:11.533+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-14T00:00:00+00:00'
[2023-10-18T11:17:11.552+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:11.557+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:11.558+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:11.558+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:11.584+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210314T000000, start_date=20231018T111711, end_date=20231018T111711
[2023-10-18T11:17:11.633+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:11.702+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:49.993+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-19T12:11:50.003+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [queued]>
[2023-10-19T12:11:50.004+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:50.021+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-14 00:00:00+00:00
[2023-10-19T12:11:50.028+0000] {standard_task_runner.py:57} INFO - Started process 394 to run task
[2023-10-19T12:11:50.031+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-14T00:00:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp4xv8q8cb']
[2023-10-19T12:11:50.033+0000] {standard_task_runner.py:85} INFO - Job 104: Subtask waiting_for_data
[2023-10-19T12:11:50.080+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-14T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:50.163+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-14T00:00:00+00:00'
[2023-10-19T12:11:50.173+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:50.173+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:50.174+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:50.174+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:50.185+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210314T000000, start_date=20231019T121149, end_date=20231019T121150
[2023-10-19T12:11:50.246+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:50.286+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
